onnxscript/converter.py

```diff
@@ -1253,7 +1251,7 @@ def translate_loop_stmt(self, loop_stmt: Union[ast.For, ast.While]) -> None:
             self.source_of(loop_stmt),
         )
         for pv in loop_state_vars:
-            ov = self.py_var_to_onnx_var(pv, self.source_of(loop_stmt))
+            ov = self.py_var_to_onnx_var(pv, self.source_of(loop_stmt)).name
```

Comment:
nit: What are ov and pv? Possible to have better names for them?

Reply:
pv stands for python-variable and ov for the corresponding onnx-variable. Let me postpone this fix to another PR (there are more occurrences like this, and the CI takes too long to run again, so merged it for now).